clang/include/clang/Basic/TargetOptions.h

```diff
@@ -83,6 +83,10 @@ class TargetOptions {
   llvm::VersionTuple SDKVersion;
 };
 
+static inline bool isCapABI(const llvm::StringRef &ABI) {
```

Comment:
1. This name sucks (hybrid is arguably a cap ABI)
2. I still believe you would be much better off reusing the purecap ABI name and keying off the OS when necessary. This avoids duplicating information so is conceptually cleaner, it makes it easier for people to write code for both CheriOS and CheriBSD (it's always `-mabi=purecap` if you want a pure capability ABI), it aligns better with RISC-V, it avoids confusion ("why can't I use the purecap ABI on the OS that's built around pure capability code!?!?!") and, finally, I believe there are many more places where you want the same behaviour as CheriBSD than places that differ.

Reply:
I think the issue here is that you can use purecap on CheriOS (or at least there is nothing stopping you), and you could also use the CheriOS specific ABI on FreeBSD. I think that, at least in principle, these are two different ABIs, that share a lot in common. I agree the name could stand to change.

Reply:
Same fix as other comment